### PR TITLE
Touch Up Container article

### PR DESCRIPTION
**Pull request by adamgordonbell**
Preview:
https://65f1beb323cfd53358988001--ephemeral-narwhal-3ac3b2.netlify.app/blog/docker-alternatives/

### CODE REVIEW

File: blog/_posts/2022-04-23-docker-alternatives.md [hunk 1]
```diff
@@ -16,21 +16,34 @@ last_modified_at: 2023-07-14
...
+| **Image Builder**   | A tool or command used to create container images from a specification, typically a Dockerfile. It compiles the application code, dependencies, and settings into a container image. |
+| **Container Runtime** | The software or platform that runs and manages containers based on container images. It handles the lifecycle of containers, including creation, execution, and termination. |
+</div>
 
-But Docker isn't your only option. As cloud tools evolve and more companies switch to containerization, more players are entering the market. Some of these are wholly different products, while others are offshoots from Docker. Understanding the pros and cons of these alternative tools can help you choose one that works for your team.
+Thankfully Docker isn't your only option for building image or for running containers. As cloud tools evolve and more companies switch to containerization, more players are entering the market. Some of these are wholly different products, while others are offshoots from Docker. Understanding the pros and cons of these alternative tools can help you choose one that works for your team.
```
Comment (idelvall) [inline blog/_posts/2022-04-23-docker-alternatives.md:30]:
```suggestion
Thankfully Docker isn't your only option for building images or for running containers. As cloud tools evolve and more companies switch to containerization, more players are entering the market. Some of these are wholly different products, while others are offshoots from Docker. Understanding the pros and cons of these alternative tools can help you choose one that works for your team.
```

File: blog/_posts/2022-04-23-docker-alternatives.md [hunk 2]
```diff
@@ -16,21 +16,34 @@ last_modified_at: 2023-07-14
...
 
-This article will examine some of the more popular alternatives to Docker, both container and non-container tools, and compare them in terms of their scalability, documentation quality, cost, and other factors.
+This article will examine some of the more popular alternatives to Docker, both as a runtime and for creating OCI Compatible images. We will even examine some non-container tools, and compare them in terms of their scalability, documentation quality, cost, and other factors.
```
Comment (idelvall) [inline blog/_posts/2022-04-23-docker-alternatives.md:32]:
```suggestion
This article will examine some of the more popular alternatives to Docker, both as a runtime and for creating OCI compatible images. We will even examine some non-container tools, and compare them in terms of their scalability, documentation quality, cost, and other factors.
```

File: blog/_posts/2022-04-23-docker-alternatives.md [hunk 3]
```diff
@@ -16,21 +16,34 @@ last_modified_at: 2023-07-14
...
+## Podman
 
-It's a great way to level up if you already work with Docker, allowing you to easily find and manage Docker containers. It also runs with other products, like [Kubernetes](https://kubernetes.io/).
+[Podman](/blog/podman-rootless) is an image builder and container runtime. It aims to be a direct replacement for Docker, and it uses the same commands, meaning you can swap it into existing projects easily.
 
-Podman can create customized versions of containers and share them with other users. If you regularly use containerized products, it lets you explore other products. If you're developing workflows, it can help you test the waters.
+It's a great way to level up if you already work with Docker, allowing you to easily find and manage Docker containers.
 
-Podman's documentation is excellent for a small-scale project and includes a well-written introduction, an API reference, and several tutorials.
+With Podman you can create images. Podman's documentation is excellent for a small-scale project and includes a well-written introduction, an API reference, and several tutorials.
```
Comment (idelvall) [inline blog/_posts/2022-04-23-docker-alternatives.md:46]: >is excellent for a small-scale project

What does this mean?

Comment (adamgordonbell) [reply blog/_posts/2022-04-23-docker-alternatives.md:46]: Even though it's a smaller and less popular project than docker, it's documentation is excellant.

**Pull request merged**